auth0/src/main/java/com/auth0/android/provider/SymmetricSignatureVerifier.java

```diff
@@ -0,0 +1,17 @@
+package com.auth0.android.provider;
+
+import android.support.annotation.NonNull;
+
+import com.auth0.android.jwt.JWT;
+
+/**
+ * Token signature verifier for HS256 algorithms.
+ */
+class SymmetricSignatureVerifier extends SignatureVerifier {
```

Comment:
Kept for similarity across implementations. But could remove this class if the main class is declared non-abstract and already implements "doing nothing". Asymmetric could subclass it and implement the new behavior, like today.

Reply:
I would suggest omitting it. Creates a bit of confusion having it (IMHO)

Reply:
@damieng what's your take here? I know for .net you've added a `NoSignatureVerifier` that extends/implements `SignatureVerifier`. Should we keep this but rename it as yours, or remove it?